recipes/file/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "file" %}
+{% set version = "5.37" %}
+{% set sha256 = "e9c13967f7dd339a3c241b7710ba093560b9a33013491318e88e6b8b57bae07f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: ftp://ftp.astron.com/pub/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

```suggestion
  sha256: e9c13967f7dd339a3c241b7710ba093560b9a33013491318e88e6b8b57bae07f
```

and you can remove that jinja variable please

---

I don't mind doing it. But why is it better to keep it here? It's super convenient to have all volatile parameters on top of the file.

---

Just to reduce the number of jinja variables, but that is fine to let that sha256 jinja variable
I usually recommend to remove it and let it just on ``sha256``especially because of the bot which will update the recipe. If you let that variable as it is the bot will also be able to update it. However, I think if you rename it, the bot will not be able to update this recipe.

But if you prefer to let that as it is, it is fine, but just remember to never change the name of that variable :p

---

Ah, I see, so it's mostly for the bot automation. As it can handle the jinja var named `sha256`, I prefer to keep it. And it's good to know the variable must be named this way. Thanks for the feedback, @marcelotrevisani!